ts/index.ts

```diff
@@ -182,6 +191,13 @@ class Gossipsub extends Pubsub {
      */
     this.direct = new Set(opts.directPeers.map(p => p.id.toB58String()))
 
+    /**
+     * Map of peer id and AcceptRequestWhileListEntry
+     *
+     * @type {Map<string, AcceptFromWhiteListEntry}
+     */
+    this.acceptFromWhitelist = new Map()
```

Comment:
I think we need a place where these entries get routinely pruned so they don't build up.
Maybe you can prune old entries on every N heartbeats

Reply:
also remove the peer's entry in `_removePeer`

Reply:
@wemeetagain  so I removed it in `_removePeer` but regarding this

> I think we need a place where these entries get routinely pruned so they don't build up.
Maybe you can prune old entries on every N heartbeats

I don't think it's necessary as we only do the prune in `_removePeer()` function for other map as well (unless there's already a leak somewhere, we'll have to do the prune regularly for all of the map). In `acceptFrom()` we also do the prune if the score is negative